Solutions/Nairo/exercise-1/app/src/handler.py

```diff
@@ -0,0 +1,26 @@
+import json
+import logging
+from app.tools.aws_utility import *
```

Comment:
Sempre que possível, importar apenas os métodos ou classes que estamos utilizando no código.

ref: https://www.geeksforgeeks.org/why-import-star-in-python-is-a-bad-idea/